backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/Element.java

```diff
@@ -294,6 +294,10 @@ public interface Element extends EObject {
     /**
      * Returns the value of the '<em><b>Qualified Name</b></em>' attribute.
      * <!-- begin-user-doc -->
+     * The full ownership-qualified name of this Element, represented in a form that is valid according to the KerML
```

Comment:
Could you please move this comment into ElementImpl.java? It is where we modify derived references methods so It make sense to have the explanation in *Impl.java methods.

Reply:
Done